.github/validate-rapps.py

```diff
@@ -39,6 +46,12 @@
 ]
 
 
+all_name_lines = []
+all_url_lines = []
+all_names = []
+all_urls = []
```

Comment:
If you make these things a dictionary, you can lookup / use them easier.
f.e.:

```
# Define:
all_names = {}
# Lookup (note the 'None' default argument, without that this throws)
first = all_names.get(name, None)
# Use it
if first is not None:
    print(f'Duplicate shit, first occurrence: {first}')
else:
    all_names[name] = line
```

Reply:
Addressed, thanks! :)